### PR TITLE
added 4th ring and expanded view space for radar

### PR DESCRIPTION
**Pull request by gm7**
Minor styling changes to get the 4 ring tech radar to display with a number of technology items (3 columns)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,18 +45,25 @@ yarn lint
 yarn build
```
Comment (melissapalmer) [inline README.md]: Is it not possible to get the Docker image built, without the need to have to install other things eg: yarn? can that not be included in the Dockerfile steps?

File: src/components/Description.jsx [hunk 2]
```diff
@@ -8,12 +8,12 @@ const Description = () => (
...
     <Typography variant="subtitle2">
-      This tool shows my skills and directions in which I intend to develop. Based on the <a href="https://www.thoughtworks.com/radar">pioneering
-      work of ThoughtWorks</a> and <a href="https://github.com/zalando/tech-radar">Zalando&#39;s project</a>.
+      This tool shows the current state of the methodologies, technologies, protocols and tools used by the Big Baobab team. This is based on the <a href="https://www.thoughtworks.com/radar">pioneering
+      work of ThoughtWorks</a>, <a href="https://github.com/zalando/tech-radar">Zalando&#39;s project</a> and <a href="https://github.com/yaneek/tech-radar">Grzegorz Marchwiński</a>.
     </Typography>
     <Typography variant="subtitle2">
       The Tech Radar is a list of technologies, methodologies,  complemented by an assessment result, called <em>ring
-        assignment</em>. I use four rings with the following semantics:
+        assignment</em>. There are four rings with the following semantics:
```
Comment (melissapalmer) [inline src/components/Description.jsx]: Is it 4 or 5 rings now if you adding one?

File: README.md (hunk 1 above)
Comment (gm7) [reply README.md]: docker build is in our downstream. This is just the "framework". The steps here are for running locally

File: src/components/Description.jsx (hunk 2 above)
Comment (gm7) [reply src/components/Description.jsx]: 5 rings

**Pull request merged**